<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="9" hostname="LAPTOP-3CF66GBG" failures="5" tests="5" name="RegressionTest(failed)" time="0.145" errors="0" timestamp="2022-09-13T19:04:59 IST">
  <testcase classname="com.satwa.tests.Contacts" name="modifyContact" time="0.035">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.satwa.tests.Contacts.modifyContact(Contacts.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- modifyContact -->
  <testcase classname="com.satwa.tests.Contacts" name="createNewCOntact" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.satwa.tests.Contacts.createNewCOntact(Contacts.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- createNewCOntact -->
  <testcase classname="com.satwa.tests.Deals" name="deleteMultipleDeals" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.satwa.tests.Deals.deleteMultipleDeals(Deals.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- deleteMultipleDeals -->
  <testcase classname="com.satwa.tests.Deals" name="createNewDeal_pastDate" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.satwa.tests.Deals.createNewDeal_pastDate(Deals.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- createNewDeal_pastDate -->
  <testcase classname="com.satwa.tests.Deals" name="deleteDeal" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.satwa.tests.Deals.deleteDeal(Deals.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- deleteDeal -->
  <testcase classname="com.satwa.tests.Contacts" name="modifyCOntact_updateStatus" time="0.0">
    <ignored/>
  </testcase> <!-- modifyCOntact_updateStatus -->
  <testcase classname="com.satwa.tests.Contacts" name="deleteMultipleContacts" time="0.0">
    <ignored/>
  </testcase> <!-- deleteMultipleContacts -->
  <testcase classname="com.satwa.tests.Contacts" name="createNewConract_activeStatus" time="0.0">
    <ignored/>
  </testcase> <!-- createNewConract_activeStatus -->
  <testcase classname="com.satwa.tests.Contacts" name="createNewDeal_duplicateInfo" time="0.0">
    <ignored/>
  </testcase> <!-- createNewDeal_duplicateInfo -->
  <testcase classname="com.satwa.tests.Deals" name="createNewDeal_futureDate" time="0.0">
    <ignored/>
  </testcase> <!-- createNewDeal_futureDate -->
  <testcase classname="com.satwa.tests.Deals" name="modifyDeal_updateStatus" time="0.0">
    <ignored/>
  </testcase> <!-- modifyDeal_updateStatus -->
  <testcase classname="com.satwa.tests.Deals" name="modifyDeal" time="0.0">
    <ignored/>
  </testcase> <!-- modifyDeal -->
  <testcase classname="com.satwa.tests.Contacts" name="deleteCOntact" time="0.0">
    <ignored/>
  </testcase> <!-- deleteCOntact -->
  <testcase classname="com.satwa.tests.Deals" name="createNewDeal" time="0.0">
    <ignored/>
  </testcase> <!-- createNewDeal -->
</testsuite> <!-- RegressionTest(failed) -->
